From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giuseppe Maggio <jertlok@proton.me>
Date: Thu, 27 Aug 2026 11:08:06 +0200
Subject: [PATCH] ASoC: tas2557: drop an unused variable in hw_params

It has never been assigned; the function returns tas2557_setup()'s result
directly. clang warns about it on every build.

Signed-off-by: Giuseppe Maggio <jertlok@proton.me>
---
 sound/soc/codecs/tas2557.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/codecs/tas2557.c b/sound/soc/codecs/tas2557.c
index 0010c91b2069..f5613783ee2a 100644
--- a/sound/soc/codecs/tas2557.c
+++ b/sound/soc/codecs/tas2557.c
@@ -1435,7 +1435,6 @@ static int tas2557_hw_params(struct snd_pcm_substream *substream,
 			     struct snd_soc_dai *dai)
 {
 	struct tas2557 *tas = snd_soc_component_get_drvdata(dai->component);
-	int ret;
 
 	tas->width = params_width(params);
 
